src/Inkluzitron/Modules/TinderVoteModule.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Discord;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using Inkluzitron.Contracts;
+using Inkluzitron.Extensions;
+using Inkluzitron.Models.Settings;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.FileSystemGlobbing.Internal.Patterns;
+
+namespace Inkluzitron.Modules
+{
+    public class TinderVoteModule : ModuleBase, IReactionHandler
```

Comment:
tohle by spíš měla být služba než modul. Imo

Reply:
+ `IRecationHandler` nedávat k `ModuleBase`

Reply:
Co se ti nelíbí na kombinaci ModuleBase a IReactionHandler?

Reply:
Ale je pravda, že se tu nepoužívá nic z ModuleBase, takže jde beztrestně vyhodit a nechat jen IReactionHandler.

Reply:
To že to moc nefunguje s těma singletonama. Ta třída se registruje jako singleton do discrod.net protože je to module a taky se udělá instance pro ten reactionhandler -> budou dvě instance úplně zbytečně. To už jsme myslím jednou řešili a byl si to zrovna ty, co mě za to posledně zjebal :kek: